doc/ledger3.texi

```diff
@@ -2919,7 +2919,7 @@ after the amount, or on its own line preceded by whitespace:
 @c > Is there a way to produce a register report that lists all the transaction
 @c > that contain a certain tag, and sort them based on the value of the tag?
 @c ledger reg --sort "tag('foo')" %foo
-@c ledger reg --group-by "tag('Employer)" Remboursement:Employer and tag Employer
+@c ledger reg --group-by "tag('Employer)" Reimbursement:Employer and tag Employer
 @c > Is it possible to get subtotals for each tag value?
```

Comment:
So maybe the fix would be changing Employer to Employeur to underline ledger's universality :)